src/Microsoft.ML.CpuMath/CpuMathUtils.netstandard.cs

```diff
@@ -15,77 +15,77 @@ public static partial class CpuMathUtils
+        public static float SumSq(float mean, float[] source, int offset, int count) => SseUtils.SumSq(mean, source, offset, count);
 
-        public static float SumAbs(float[] src, int count) => SseUtils.SumAbs(src, count);
+        public static float SumAbs(float[] source, int count) => SseUtils.SumAbs(source, count);
 
-        public static float SumAbs(float[] src, int offset, int count) => SseUtils.SumAbs(src, offset, count);
+        public static float SumAbs(float[] source, int offset, int count) => SseUtils.SumAbs(source, offset, count);
 
-        public static float SumAbs(float mean, float[] src, int offset, int count) => SseUtils.SumAbs(mean, src, offset, count);
+        public static float SumAbs(float mean, float[] source, int offset, int count) => SseUtils.SumAbs(mean, source, offset, count);
 
-        public static float MaxAbs(float[] src, int count) => SseUtils.MaxAbs(src, count);
+        public static float MaxAbs(float[] source, int count) => SseUtils.MaxAbs(source, count);
 
-        public static float MaxAbs(float[] src, int offset, int count) => SseUtils.MaxAbs(src, offset, count);
+        public static float MaxAbs(float[] source, int offset, int count) => SseUtils.MaxAbs(source, offset, count);
 
-        public static float MaxAbsDiff(float mean, float[] src, int count) => SseUtils.MaxAbsDiff(mean, src, count);
+        public static float MaxAbsDiff(float mean, float[] source, int count) => SseUtils.MaxAbsDiff(mean, source, count);
 
-        public static float DotProductDense(float[] a, float[] b, int count) => SseUtils.DotProductDense(a, b, count);
+        public static float DotProductDense(float[] value, float[] b, int count) => SseUtils.DotProductDense(value, b, count);
 
-        public static float DotProductDense(float[] a, int offset, float[] b, int count) => SseUtils.DotProductDense(a, offset, b, count);
+        public static float DotProductDense(float[] value, int offset, float[] b, int count) => SseUtils.DotProductDense(value, offset, b, count);
 
-        public static float DotProductSparse(float[] a, float[] b, int[] indices, int count) => SseUtils.DotProductSparse(a, b, indices, count);
+        public static float DotProductSparse(float[] value, float[] b, int[] indices, int count) => SseUtils.DotProductSparse(value, b, indices, count);
 
-        public static float DotProductSparse(float[] a, int offset, float[] b, int[] indices, int count) => SseUtils.DotProductSparse(a, offset, b, indices, count);
+        public static float DotProductSparse(float[] value, int offset, float[] b, int[] indices, int count) => SseUtils.DotProductSparse(value, offset, b, indices, count);
 
-        public static float L2DistSquared(float[] a, float[] b, int count) => SseUtils.L2DistSquared(a, b, count);
+        public static float L2DistSquared(float[] value, float[] b, int count) => SseUtils.L2DistSquared(value, b, count);
```

Comment:
`b` should be updated as well.  Similar comment throughout this code file.

Reply:
Actually, I don't think `value` makes sense for the first parameter either.

---
In reply to: [221718385](https://github.com/dotnet/machinelearning/pull/1021#discussion_r221718385) [](ancestors = 221718385)

Reply:
Would `source` and `destination` make sense there?

Reply:
Not for the specific method, no. This is computing the distance between two arrays and returning a `float`. So there isn't a `destination` array.

`left` and `right` would make sense.  Or `a` and `b`... but I think you are changing it to use `left` and `right` elsewhere, so let's be consistent.